src/main/java/net/comments/ft/Test4.java

```diff
@@ -0,0 +1,29 @@
+package net.comments.ft;
+
+import net.comments.objects.BCommentPage;
+import net.comments.objects.Comment;
+import net.comments.objects.CommentPage;
+import net.comments.objects.CommentTable;
+import org.hamcrest.MatcherAssert;
+import org.testng.annotations.Test;
+
+public class Test4 extends BaseDriver {
+    private final CommentPage page;
+
+    public Test4() {
+        super();
+        this.page = new BCommentPage(this.driver);
+    }
+
+    @Test
+    public void test() {
+        int commentID = 1;
+        CommentTable commentTable = page.getCurrentComments();
+        page.open();
```

Comment:
As I understand the test, first the page has to be open. And then, the interactions with the `CommentTable` have to happen.